rsmtool/notebooks/summary/evaluation.ipynb

```diff
@@ -37,44 +40,53 @@
+    "            # finally, we calculate the max and min scores; if we couldn't get any trim values,\n",
+    "            # then we default these to `?` and the set `has_missing_trims=True`\n",
+    "            if trim_min is None:\n",
+    "                min_score, has_missing_trims = '?', True\n",
     "            else:\n",
-    "                postproc_file = os.path.join(csvdir, '{}_postprocessing_params.{}'.format(model_id, file_format))\n",
-    "                if os.path.exists(postproc_file):\n",
-    "                    df_postproc = DataReader.read_from_file(postproc_file)\n",
-    "                    min_score = df_postproc['trim_min'].values[0] - trim_tolerance\n",
-    "                    max_score = df_postproc['trim_max'].values[0] + trim_tolerance\n",
-    "                else:\n",
-    "                    display(Markdown('**Note:** Minimum and maximum truncation scores could '\n",
-    "                                     'be located for `{}`. Truncation range values will be '\n",
-    "                                     'set to `?`.'.format(model_id)))\n",
-    "                    min_score, max_score = '?', '?'\n",
+    "                min_score = float(trim_min) - config.get('trim_tolerance', 0.49998)\n",
+    "            if trim_max is None:\n",
+    "                max_score, has_missing_trims = '?', True\n",
+    "            else:\n",
+    "                max_score = float(trim_max) + config.get('trim_tolerance', 0.49998)        \n",
     "\n",
     "            df_eval['truncation range'] = \"[{}, {}]\".format(min_score, max_score)\n",
     "            \n",
-    "            #rename the columns to remove reference to scale/raw scores\n",
+    "            # rename the columns to remove reference to scale/raw scores\n",
     "            new_column_names = [col.split('.')[0] if not 'round' in col \n",
     "                                else '{} (rounded)'.format(col.split('.')[0])\n",
     "                                for col in df_eval.columns ]\n",
     "            df_eval.columns = new_column_names\n",
-    "            evals.append(df_eval)          \n",
+    "            evals.append(df_eval)\n",
+    "\n",
     "    if len(evals) > 0:\n",
     "        df_evals = pd.concat(evals, sort=True)\n",
     "    else:\n",
     "        df_evals = pd.DataFrame()\n",
-    "    return(df_evals)\n",
+    "    return df_evals, has_missing_trims\n",
+    "\n",
+    "df_eval, has_missing_trims = read_evals(model_list, file_format_summarize)\n",
     "\n",
-    "df_eval = read_evals(model_list, file_format_summarize)\n",
+    "if has_missing_trims:\n",
+    "    display(Markdown('**Note:** Minimum and maximum truncation scores could not '\n",
+    "                     'be located in some cases. When truncation values are missing, '\n",
+    "                     'they will be set to `?`.'))\n",
```

Comment:
1. I think we need to say _why_ they could not be located. 
2. Are the terms "Minimum and maximum truncation scores" and "truncation values" referring to the same entities? If so, perhaps a better name for said entities would be something like "The minimum and maximum scores after truncation"?